Sources/Networking/HTTPClient/HTTPStatusCode.swift

```diff
@@ -78,7 +78,7 @@ extension HTTPStatusCode: Codable {}
 extension HTTPStatusCode {
 
     var isSuccessfulResponse: Bool {
-        return 200...299 ~= self.rawValue
+        return 200...399 ~= self.rawValue
```

Comment:
Changed this to match Android.
I confirmed nothing else uses this method, and the only other usage is unaffected.